webapp/src/components/conference/index.ts

```diff
@@ -1,17 +1,16 @@
 import {connect} from 'react-redux';
 import {bindActionCreators, Dispatch} from 'redux';
 
-import {GlobalState} from 'mattermost-redux/types/store';
 import {GenericAction} from 'mattermost-redux/types/actions';
 
-import {id as pluginId} from '../../manifest';
+import {GlobalState} from '../../types';
 import Conference from './conference';
 import {openJitsiMeeting} from '../../actions';
 
 function mapStateToProps(state: GlobalState) {
     return {
-        post: (state as any)[`plugins-${pluginId}`].openMeeting,
-        jwt: (state as any)[`plugins-${pluginId}`].openMeetingJwt
+        post: state['plugins-jitsi'].openMeeting,
+        jwt: state['plugins-jitsi'].openMeetingJwt
```

Comment:
Why using directly `jitsi` instead of `pluginId`?

Reply:
If we want to have strong typed check for the state, this is the way.

Reply:
Thanks! I am 0/5 on whether to keep the strong typing, or the variability of the plugin ID. I let the final decision to you.